### PR TITLE
Update limits-specifications-teams.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6104
Added note.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/limits-specifications-teams.md [hunk 1]
```diff
@@ -132,6 +132,9 @@ Channel names also can't start with an underscore (_) or period (.), or end with
 
 ### Meeting expiration
 
+> [!NOTE]
+> A meeting URL will never stop working. The expiry only relates to any PSTN dial-in number and/or underlying meeting policies and settings.
```
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:136]:
```suggestion
> A meeting URL will never stop working. The expiry only relates to any PSTN dial-in numbers and/or underlying meeting policies and settings.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**